<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWidget</class>
 <widget class="QWidget" name="MusicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabTrack">
      <attribute name="title">
       <string>Track</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>546</width>
            <height>265</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="editTrackName"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Number:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KIntNumInput" name="editTrackNumber"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Release Date:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="editTrackReleaseDate"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Performer:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="editTrackPerformer"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Genre:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KLineEdit" name="editTrackGenre"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>See Also:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="KLineEdit" name="editTrackSeeAlso"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Lyrics:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="KTextEdit" name="editTrackLyrics"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAlbum">
      <attribute name="title">
       <string>Album</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>546</width>
            <height>265</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelAlbumPerformer">
             <property name="text">
              <string>Performer:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="editAlbumPerformer"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelAlbumName">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="editAlbumName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelAlbumCover">
             <property name="text">
              <string>Cover:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="editAlbumCover"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelAlbumTracks">
             <property name="text">
              <string>Tracks:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KIntNumInput" name="editAlbumTracks"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
